File: Manhattan College App/ViewController.m
//
//  ViewController.m
//  Manhattan College App
//
//  Created by Nick Monaco on 2/3/14.
//  Copyright (c) 2014 Monaco Software Productions. All rights reserved.
//

#import "ViewController.h"
#import "HWTableViewController.h"

@interface ViewController ()

@end

@implementation ViewController

-(IBAction)facebook
{
    [[UIApplication sharedApplication] openURL:[NSURL URLWithString:@"https://www.facebook.com/ManhattanCollegeEdu"]];
}

-(IBAction)twitter
{
    [[UIApplication sharedApplication] openURL:[NSURL URLWithString:@"https://twitter.com/MCStudAct"]];
}

-(IBAction)instagram
{
    [[UIApplication sharedApplication] openURL:[NSURL URLWithString:@"https://instagram.com/ManhattanEdu"]];
}

-(IBAction)switchController:(id)sender
{
   // NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
    //NSMutableArray *loadData = [defaults objectForKey:@"saveData"];
    
    
    
    
    [self performSegueWithIdentifier:@"HW" sender:self];
}

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
    /*
    [self.scrollView setScrollEnabled:YES];
    [self.scrollView setContentSize:(CGSizeMake(320, 535))];
     */
}

- (void)viewDidAppear:(BOOL)animated {
    [self.scrollView setScrollEnabled:YES];
    [self.scrollView setContentSize:(CGSizeMake(320, 537))];
    
    //self.screenName = @"Home screen";
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Settings.bundle/Manhattan College App/ViewController.swift
////
////  ViewController.swift
////  MC
////
////  Created by Nick Monaco on 9/22/16.
////  Copyright © 2016 Monaco Software Productions. All rights reserved.
////
//
//import Foundation
//import UIKit
//
//class ViewController: UIViewController {
//    @IBOutlet var scrollView: UIScrollView!
//    
//    @IBAction func facebook() {
//        UIApplication.shared.openURL(NSURL(string: "https://www.facebook.com/ManhattanCollegeEdu")! as URL)
//    }
//    
//    @IBAction func twitter() {
//        UIApplication.shared.openURL(NSURL(string: "https://twitter.com/MCStudentEngage")! as URL)
//    }
//    
//    @IBAction func instagram() {
//        UIApplication.shared.openURL(NSURL(string: "https://instagram.com/ManhattanEdu")! as URL)
//    }
//    
//    @IBAction func more() {
//        UIApplication.shared.openURL(NSURL(string: "https://inside.manhattan.edu")! as URL)
//    }
//    
//    @IBAction func `switch`(_ sender: Any) {
//        // NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
//        //NSMutableArray *loadData = [defaults objectForKey:@"saveData"];
//        self.performSegue(withIdentifier: "HW", sender: self)
//    }
//    
//    override func viewDidLoad() {
//        super.viewDidLoad()
//        // Do any additional setup after loading the view, typically from a nib.
//        /*
//         [self.scrollView setScrollEnabled:YES];
//         [self.scrollView setContentSize:(CGSizeMake(320, 535))];
//         */
//    }
//    
//    override func viewDidAppear(_ animated: Bool) {
//        self.scrollView.isScrollEnabled = true
//        self.scrollView.contentSize = (CGSize(width: 320, height: 537))
//        //self.screenName = @"Home screen";
//    }
//    
//    override func didReceiveMemoryWarning() {
//        super.didReceiveMemoryWarning()
//        // Dispose of any resources that can be recreated.
//    }
//}
